crates/utils/src/signing.rs

```diff
@@ -46,12 +46,13 @@ pub fn verify_signed_commit_boost_message<T: HashTreeRoot>(
     message: &mut T,
     signature: &BlsSignature,
     public_key: &BlsPublicKey,
-    slot_hint: Option<Slot>,
+    _slot_hint: Option<Slot>,
```

Comment:
remove it if it is not neccesary.

Reply:
it's a temporary fix, other helix verification method use slot.

Reply:
No. The original algorithm doesn't use slot.

Reply:
The way you see using slot is for block signing